lib/elixir/lib/inspect/algebra.ex

```diff
@@ -303,19 +305,22 @@ defmodule Inspect.Algebra do
 
   """
   @spec nest(t, non_neg_integer) :: doc_nest
-  def nest(x, 0) when is_doc(x) do
-    x
+  def nest(doc, positions)
+
+  def nest(doc, 0) when is_doc(doc) do
+    doc
   end
 
-  def nest(x, i) when is_doc(x) and is_integer(i) do
-    doc_nest(x, i)
+  def nest(doc, positions)
+      when is_doc(doc) and is_integer(positions) and positions > 0 do
```

Comment:
I think the style we generally use in Elixir is to have the operator in the previous line, because it is clear it will continue on the next one. I think the only exception is the pipe op.